### ISSUE TITLE
need more info in the report view

### ISSUE THREAD

**Issue by lisabettefay** [CAVaccineInventory/vial#487]
WB/QA often need to reference additional information from the location when creating/viewing reports.

Could the following fields be added (as indicated in the mock up below, preferably in the order shown):
- `full address`
- `location type`
- `phone number`

Also, could `Location:` be changed to display `Location name:` And could the date be formatted to split in to two lines and be formatted as follows (removing the ordinal number and replacing it with an integer):

`1st May 2021 6:11:45 PM PDT`
becomes
`1 May 2021`
`6:11 PM PDT`

Current info at the top of a record:
![image](https://user-images.githubusercontent.com/81186147/116798746-e8234380-aac0-11eb-8807-21ae92cbad4b.png)

Proposed info/mock up:
![image](https://user-images.githubusercontent.com/81186147/116798735-c1650d00-aac0-11eb-935f-7acfead4b6fe.png)

**Comment by ugotsoul** [comment]
@lisabettefay the last part of your request I can't implement without removing the timezone (it breaks timezone localization for the user).

Would this be okay?
```
1 May 2021
6:11 PM
```

**Comment by ugotsoul** [comment]
@lisabettefay this was deployed today!

**Comment by ugotsoul** [comment]
@lisabettefay if you want the location link to have the address too, should we remove the full address field? it seems redundant.

![image](https://user-images.githubusercontent.com/6485006/124338858-efb5a600-db5e-11eb-89d1-6545f759b23b.png)

**Comment by lisabettefay** [comment]
YES i would like the name and full address to go under the reporter and then the location type and what ever bonus field i n the middle. please. @ugotsoul

**Comment by ugotsoul** [comment]
so like: 

reporter | website + tel | public id 
location | location type | created at

**Comment by lisabettefay** [comment]
> so like:
> 
> reporter | website + tel | public id
> location | location type | created at

this would be AWESOME.

**Comment by ugotsoul** [comment]
@lisabettefay sorry for the delay! is the ph# based on the location?
